lore/end_users/use/README.md

```diff
@@ -34,3 +35,41 @@ Now we have a pull request that has core contributor approval and successful che
 This image illustrates the BonnyCI continuous integration pipeline.
 
 ![BonnyCI Workflow](../../misc/images/BonnyCIWorkflow.png)
+
+## Handling Merge Failures
+
+Anne-Bonny, the bot user which handles checking and gating pull requests, may
+report back to a pull request informing the developer and reviewers that she
+encountered a merge failure. This typically happens in the gate queue, and will
+usually fall into one of the two categories detailed below.
+
+### Stale Pull Requests
+
+Occasionally, a pull request may fall by the wayside while other changes are
+given a higher priority by reviewers. This can leave behind a pull request
+which has become stale, meaning that the changes it is making are based off of
+an older git history. In most cases, this can be resolved with a simple rebase:
+
+```shell
```

Comment:
nit: While this shell could potentially work for some, I think most people will have origin set to the remote that the PR should be merged into rather than their personal fork of the repo.